### DISCUSSION TITLE
[Mod Idea] Self Regnerating Power Cell

### DISCUSSION THREAD

**Discussion by ApolloX939** [colobot/colobot#1816] [Ideas]
Something like, a small colony of micro biological organisms which produce their own food and produce a slight amount of power.

So it's like a Power Cell except it recharges slowly by itself. (Or you could also use the Power Station too)

> Of course I'd make it unlock-able via the Research Center near the level which would unlock the Nuclear Cell. (maybe include it at the same time as the Nuclear Cell)

I would make it so it has 70-80% the normal capacity of the Power Cell and recharges say 0.5-2.5% it's capacity per second (or maybe more percent recharged but every say 2-3 seconds)

> I'm not sure where to begin with this, most of this is just asking for some kind of documentation on how I could create a mod which adds a new item in this case something like a Bio Cell.

**Comment by esylm** [comment]
I actually really like the idea of the bio cell. But I don't really know how to create a mod to add this great item. I think you need to download the source code and then edit it. [Ask this guy](https://www.reddit.com/r/Colobot/comments/expwka/this_game_needs_some_more_content_source_code_mod/) I commented on this guys post on the colobot subredit. He might know better and help you probably

**Comment by immibis** [comment]
@esylm That is me. I think this is a very sensible idea for a mod and also possible to make. Of course, someone has to actually do it.

Maybe you have to put alien matter in the power cell factory.
